### DISCUSSION TITLE
CWSM server

### DISCUSSION THREAD

**Discussion by DarthDiamond** [nonamecrackers2/crackers-wither-storm-mod#2292] [Q&A] [answered]
Im wondering if anyone was thinking of starting a server with this mod and also maybe a official server from the creators of this mod

**Comment by nonamecrackers2** [comment, marked as answer]
There won't be an official server, but you can join the Discord server and see if anyone is hosting a server there: https://discord.gg/cracker-s-modded-community-987817685293355028
